### PR TITLE
OC-140: Implementing new UI of Add Address screen, Edit Address and my Address.

### PR DESCRIPTION
**Pull request by AakanshaThakur**

### CODE REVIEW

File: lib/Screens/address/widget/address_filterchip_widget.dart [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                shape: const StadiumBorder(
+                    side: BorderSide(color: AppColor.primary)),
+                backgroundColor: selectedType == item
+                    ? AppColor.primaryLight
+                    : AppColor.white,
+                labelPadding: const EdgeInsets.all(xxTiniestSpacing),
+                selectedColor: AppColor.primary,
+                onSelected: (bool value) {
+                  singleSelect(item);
+                  log('onTap===============>$selectedType');
```
Comment (Aditi-005) [inline lib/Screens/address/widget/address_filterchip_widget.dart]: remove all log

File: lib/Screens/orders/widgets/item_orders_expansiontile.dart [hunk 2]
```diff
@@ -1,10 +1,10 @@
 // import 'package:flutter/material.dart';
```
Comment (Aditi-005) [inline lib/Screens/orders/widgets/item_orders_expansiontile.dart]: remove commented code or delete this file

File: lib/blocs/address_bloc/address_bloc.dart [hunk 3]
```diff
@@ -60,15 +59,15 @@ class AddressBloc extends Bloc<AddressEvents, AddressStates> {
       AddAddress event, Emitter<AddressStates> emit) async {
     emit(AddAddressLoading());
     // try {
-    // String userId= _customerCache.g
+    log('address post===========>${event.addAddress}');
```
Comment (Aditi-005) [inline lib/blocs/address_bloc/address_bloc.dart]: remove all logs and commented code

**Review by Aditi-005 — CHANGES_REQUESTED** [review]

File: lib/Screens/address/widget/address_form.dart [hunk 4]
```diff
@@ -13,150 +17,128 @@ class AddressForm extends StatelessWidget {
...
+                        } else if (RegExp(r"^\([A-Za-z]+[A-Za-z]")
+                            .hasMatch(value)) {
+                          return 'Please Enter Valid Address';
+                        }
+                        return null;
+                      },
+                      initialValue: saveAddress['address_name'],
+                      onTextFieldChanged: (value) {
+                        saveAddress['address_name'] = value;
+                        log('onChanged=========>${saveAddress['address_name'].toString()}');
```
Comment (404ajayreddy) [inline lib/Screens/address/widget/address_form.dart]: remove logs

**Review by 404ajayreddy — CHANGES_REQUESTED** [review]

**Review by 404ajayreddy — APPROVED** [review]

**Pull request merged**